gensim/models/doc2vec.py

```diff
@@ -808,6 +809,50 @@ def delete_temporary_training_data(self, keep_doctags_vectors=True, keep_inferen
         if self.docvecs and hasattr(self.docvecs, 'doctag_syn0_lockf'):
             del self.docvecs.doctag_syn0_lockf
 
+    def save_word2vec_format(self, fname, doc_vec=True, word_vec=False, prefix='dt_', fvocab=None, binary=False):
```

Comment:
To be consistent with behavior before this change, default should be to just write word-vectors (as when implementation just inherited from Word2Vec). Let the new capability require explicit activation with a new parameter.